api/inventorypb/agent_status.proto

```diff
@@ -11,12 +11,14 @@ enum AgentStatus {
   STARTING = 1;
   // Agent is running.
   RUNNING = 2;
-  // Agent encountered error and will be restarted automatically soon.
+  // Agent will be restarted automatically soon.
   WAITING = 3;
   // Agent is stopping.
   STOPPING = 4;
   // Agent finished.
   DONE = 5;
   // Agent is not connected, we don't know anything about it's state.
   UNKNOWN = 6;
+  // Agent encountered error when starting.
+  INITIALIZATION_ERROR = 7;
```

Comment:
can we move it right after starting, so it's visible that it might be caused during starting. number still should be 7